opentelemetry-sdk/src/opentelemetry/sdk/_metrics/measurement.py

```diff
@@ -15,10 +15,12 @@
 from dataclasses import dataclass
 from typing import Union
 
+from opentelemetry._metrics.instrument import Instrument
 from opentelemetry.util.types import Attributes
 
 
 @dataclass(frozen=True)
 class Measurement:
     value: Union[int, float]
+    instrument: Instrument
```

Comment:
Is this what would differentiate this class from the API `Measurement`/`Observation` class? I think I understood in the SIG "context" would be the different attribute.

Reply:
I thought the use of word context meant as in circumstances the class being used.

Reply:
It will actually need both once we have implemented the spec. It will need the OTel Context for exemplars (and pulling extra attributes from baggage which was recently removed from the spec).